src/UnitTest/test_quote_prov.cpp

```diff
@@ -509,6 +526,17 @@ static void GetVerificationCollateralTest()
         &collateral);
     ASSERT_TRUE(SGX_QL_SUCCESS == result);
     VerifyCollateral(collateral);
+
+    collateral = nullptr;
+    result = sgx_ql_get_quote_verification_collateral_with_params(
```

Comment:
Instead of adding it part of the same test, could make a separate unit test, which calls collateral with params for both V2 and V3 collateral.